rest_framework_inclusions/renderer.py

```diff
@@ -61,17 +62,30 @@ def _render_inclusions(self, data, renderer_context):
                     continue
                 render_data[key] = value
 
+        # map the meta information, if any
+        render_data[self.response_data_key] = serializer_data
+        render_data[self.response_inclusions_key] = inclusions
+
         return render_data
 
     def render(self, data, accepted_media_type=None, renderer_context=None):
         render_data = self._render_inclusions(data, renderer_context)
-        if not render_data:
+        allowed_actions = getattr(
+            renderer_context["view"], "include_allowed_actions", None
+        )
+        if not render_data or (
+            allowed_actions and renderer_context["view"].action not in allowed_actions
+        ):
```

Comment:
I don't think we can rely on `view.action`, since non-viewset views don't have an action. Maybe a pattern like this would be better suited with sane defaults:

```py
allowed_check = getattr(renderer_context["view"], "include_allowed", default=lambda: True)
skip_includes = not allowed_check()
```

so then on the view/viewset class you can implement `def include_allowed(self):` which can look at `self.action` or `self.method` etc.